from . import base

class ArgumentError(base.BaseException): pass
